config/foreman.hiera/family/RedHat-8.yaml

```diff
@@ -0,0 +1,7 @@
+---
+# EL8 doesn't have support for SSLv3 anymore and errors out on it. This
```

Comment:
wow, how annoying.

Reply:
I must admit I didn't test it myself on EL8, but I coped this from https://github.com/puppetlabs/puppetlabs-apache/blob/624330496a431dad03e430b8f2c9ac1914c3b8de/manifests/params.pp#L821-L825